pype/plugins/hiero/publish/collect_reviews.py

```diff
@@ -205,3 +231,25 @@ def version_data(self, instance):
         instance.data["versionData"] = version_data
 
         instance.data["source"] = instance.data["sourcePath"]
+
+    def detect_sequence(self, file):
+        """ Get identificating pater for image sequence
+
+        Can find file.0001.ext, file.%02d.ext, file.####.ext
+
+        Return:
+            string: any matching sequence patern
+            int: padding of sequnce numbering
+        """
+        foundall = re.findall(r"(#+)|(%\d+d)|[^a-zA-Z](\d+)\.\w+$", file)
```

Comment:
I would just keep only `(#+)|(%\d+d)` regex as `[^a-zA-Z](\d+)\.\w+$` also match to `Asset_v001.exr` and file having frame in name does not mean it is a sequence.

Reply:
this should deal with the v### issue.. `(?<=[^a-zA-Z0-9])(\d+)(?=\.\w+$)`. Thanks ;)

Reply:
I think only `(#+)|(%\d+d)` is right. You just want to check if
- `####` is in filename (with variants e.g. `###`)
- `%04d` is in filename (with variants e.g. `%03d`)

Only in these cases it is sequence.

If the filename is `asset_v001.1001.exr` then it is not sequence but single file, right?